src/Adapter/Cache/Clearer/Symfony/ExecKernelCacheClearer.php

```diff
@@ -92,7 +92,8 @@ protected function warmUpCache(AppKernel $kernel, string $environment): bool
 
     protected function execCommand(AppKernel $kernel, string $command, string $successMessage, $errorMessage): bool
     {
-        $commandLine = 'php -d memory_limit=-1 ' . $kernel->getProjectDir() . '/bin/console ' . $command;
+        $phpPath = $_ENV['PS_PHP_BIN_PATH'] ?? 'php';
```

Comment:
```suggestion
        $phpPath = getenv('PS_PHP_BIN_PATH') ?: 'php';
```

What do you think?

Reply:
I actually prefer using `getenv()`. Thanks for the suggestion.